pkg/processor/runtime/golang/runtime.go

```diff
@@ -128,3 +128,8 @@ func (g *golang) callEntrypoint(event nuclio.Event, functionLogger logger.Logger
 
 	return
 }
+
+func (g *golang) Stop() error {
```

Comment:
This should be in AbstractRuntime - remove from golang, pypy and shell

Reply:
Will do